tools/rng_test/README.md

```diff
@@ -0,0 +1,187 @@
+# RNG(Random Number Generator) duplication testing tool
+
+This tool can be used to find duplicate random numbers generated by auth_generate_random() function.
+
+It repeats the following two operations.
+
+1. Call auth_generate_random() (executed by 'security_api' TASH command)
+2. Reboot
+
+
+
+### Prerequisites
+
+* Python 2.7 & 3.x
+
+* Install 'pySerial'
+
+  ```
+  ~$ pip install pyserial
+  ```
+
```

Comment:
Please add any CONFIG related prerequisites if any.

Reply:
I already add CONFIG related prerequisites in "How to use". Thank you.